ddtrace/internal/remoteconfig/client.py

```diff
@@ -149,6 +147,26 @@ def _load_json(data):
 TargetsType = Dict[str, ConfigMetadata]
 
 
+class RemoteConfigCallBack(six.with_metaclass(abc.ABCMeta)):
+    @abc.abstractmethod
+    def __call__(self, metadata, config):
+        # type: (Optional[ConfigMetadata], Any) -> None
+        pass
+
+
+class RemoteConfigCallBackAfterMerge(RemoteConfigCallBack):
+    configs = {}  # type: Dict[str, Any]
```

Comment:
Should the config dictionary be shared across instances?

Reply:
No it doesn't. My previous approach of this OR had got this problem in tests. With this class, everything is fine :upside_down_face: